.azure-pipelines-templates/ninja.yml

```diff
@@ -1,4 +1,10 @@
 steps:
+
+# TODO: Temporary debugging feature - show the commands for an enclave so we can confirm LVI mitigations
```

Comment:
Have we checked the binaries built with the mitigations do contain a number of additional lfences as expected?

Reply:
I hadn't, and looking at them now the `-lvi-cfg` variants are actually consistently _smaller_ than the unmitigated versions, which doesn't sound right.

```
-rw-r--r-- 1 root root  1057412 Jul 21 08:57 liboeenclave-lvi-cfg.a
-rw-r--r-- 1 root root  1363660 Jul 21 08:59 liboeenclave.a
```

Also `liboesnmalloc.a` and `liboesnmalloc-lvi-cfg.a` are exactly the same file. I'll go look at the OE build steps again and see what's going on here.

Do you have a link to that assembly-instruction-counting suggestion?

Reply:
I can compare the lfence counts in our enclave binaries:

```
edashton@edashton2:/data/src/2.CCF$ objdump -d ../CCF/build/libsmallbank.enclave.so | grep lfence | wc -l
22
edashton@edashton2:/data/src/2.CCF$ objdump -d build.lvi/libsmallbank.enclave.so | grep lfence | wc -l
7424
```

But objdump has a bunch of errors trying to parse the `-lvi-cfg.a` OE libs. I've raised this and the size discrepancy with OE.

Reply:
I was thinking of something along those lines:

```
$ objdump -d liblogging.virtual.so | grep -c cpuid
129
```

Reply:
For reference: https://reviews.llvm.org/D75939